src/test/java/bt/TestWalett.java

```diff
@@ -0,0 +1,37 @@
+package bt;
+
+import lombok.Data;
+import org.apache.commons.lang3.RandomStringUtils;
+import signumj.crypto.SignumCrypto;
+import signumj.entity.SignumAddress;
+import signumj.entity.SignumValue;
+import signumj.entity.response.Account;
+
+@Data
+public class TestWalett extends BT{
+	private SignumAddress address;
+	private String passphrase;
+
+	public TestWalett(double initialAmount) {
+		String randomPassForInitialFunding = RandomStringUtils.random(49);
+		forgeBlock(randomPassForInitialFunding);
+		// Create a random passphrase, so test can be run multiple times without
+		// rebooting the test server
+		this.passphrase = RandomStringUtils.random(49);
+		this.address = SignumCrypto.getInstance().getAddressFromPassphrase(passphrase);
+		sendAmount(randomPassForInitialFunding, this.address, SignumValue.fromSigna(initialAmount));
```

Comment:
There is no guarantee here that the sending account will have suck balance, in case initialAmount is too large.

Reply:
The solution could be to loop on forgeblock with a estimation of reward to be over the required amount. I ll do that.